demos/imagenet_classification_demo/grid_mat.hpp

```diff
@@ -0,0 +1,176 @@
+// Copyright (C) 2018-2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <algorithm>
+#include <set>
+#include <string>
+#include <vector>
+#include <queue>
+
+#include <monitors/presenter.h>
+#include <opencv2/imgproc.hpp>
+#include <opencv2/core/core.hpp>
+
+class GridMat {
+public:
+    explicit GridMat(int inputImgsCount,
+                     Presenter& presenter,
+                     const cv::Size maxDisp = cv::Size{1920, 1080},
+                     const cv::Size aspectRatio = cv::Size{16, 9},
+                     double targetFPS = 60
+                     ):
+                     size{std::max(static_cast<int>(std::ceil(targetFPS / aspectRatio.height)),
+                                   static_cast<int>(std::ceil(
+                                       1. * FLAGS_b / (aspectRatio.width + aspectRatio.height) * aspectRatio.width))
+                                    ),
+                          std::max(static_cast<int>(std::ceil(targetFPS / aspectRatio.width)),
+                                   static_cast<int>(std::ceil(
+                                       1. * FLAGS_b / (aspectRatio.width + aspectRatio.height) * aspectRatio.height))
+                                    )},
+                     currSourceID{0} {
+        int minCellSize = std::min(maxDisp.width / size.width, maxDisp.height / size.height);
+        cellSize = cv::Size(minCellSize, minCellSize);
+        rectangleHeight = presenter.graphSize.height;
+
+        for (int i = 0; i < size.width; i++) {
+            for (int j = 0; j < size.height; j++) {
+                cv::Point p;
+                p.x = cellSize.width * i;
+                p.y = rectangleHeight + (cellSize.height * j);
+                points.push_back(p);
```

Comment:
You can do it in one line: `points.emplace_back(cellSize.width * i, rectangleHeight + (cellSize.height * j));`

Reply:
Why not `emplace_back`?